lib/slim_fast/compiler.ex

```diff
@@ -9,15 +9,22 @@ defmodule SlimFast.Compiler do
 
   defp render_attribute(_, []), do: ""
   defp render_attribute(_, ""), do: ""
+  defp render_attribute(name, {:eex, opts}) do
+    value = opts[:content]
+    case value do
+      "true"  -> name
+      "false" -> ""
+      "nil"   -> ""
+      _       -> "#{name}=<%=#{value}%>"
```

Comment:
Good catch.  Unfortunately, I don't think it's as easy as `"#{name}=\"<%=#{value}%>\""`; EEx can return non-string values and HTML supports non-quoted attributes: https://html.spec.whatwg.org/multipage/syntax.html#syntax-attributes

I can see that your `parse_eex_string/1` function has been called and appropriately escaped everything, transforming `"\#{b} a"`to `"\"\#{b} a\""`.  I'll continue to dig in.